dll/cpl/powercfg/powershemes.c

```diff
@@ -543,6 +541,9 @@ DelScheme(
     if (iCurSel == CB_ERR)
         return FALSE;
 
+	if (iCurSel == 0) //as per https://jira.reactos.org/browse/CORE-16893
```

Comment:
Could you put a define instead of this magic "0" ? Also, indentation issues as usual (amend your commit instead of creating a new "formatting fixes" please)

Reply:
Magic "0", as indicated in ticket is because the only "non-deletable" item is the Home/desk profile, which is listed first in ROS as per hivedef.inf POWERCFG_NAME_0 definition (while it may not be the first in XP.

A more robust, but more complex solution would be to compare selected items's caption with POWERCFG_NAME_0

Reply:
Should this be indicated in code comments ?

Reply:
Test build in progress...

Reply:
Test failed. Please wait for next commit.

Reply:
Tested and 9dc2847 fixes everything. OK for me to go to review.